config/environments/development.rb

```diff
@@ -15,12 +14,13 @@
 
   # Enable/disable caching. By default caching is disabled.
   # Run rails dev:cache to toggle caching.
-  if Rails.root.join('tmp', 'caching-dev.txt').exist?
+
+  if Rails.root.join("tmp", "caching-dev.txt").exist?
     config.action_controller.perform_caching = true
 
     config.cache_store = :memory_store
     config.public_file_server.headers = {
-      'Cache-Control' => "public, max-age=#{2.days.to_i}"
+      "Cache-Control" => "public, max-age=#{2.days.to_i}",
```

Comment:
comma at the end is a syntax error, must be ruby swallows it

Reply:
Possibly. It came from the master branch. I'll remove it and push again.

Reply:
Line 23 is now fixed